src/components/SubTaskList/SubTaskProgressBar/SubTaskProgressBar.jsx

```diff
@@ -1,36 +1,25 @@
 import React, { Component } from 'react';
 import { Row, Col, ProgressBar } from 'react-bootstrap';
+import getProgressBarVariant from './getProgressBarVariant';
 
-class SubTaskAdd extends Component {
+class SubTaskProgressBar extends Component {
   render() {
-    const { subtaskList } = this.props;
+    const subtaskList = this.props.subtaskList || [];
```

Comment:
You can still use destructuring and assign a default value to a variable. Here is a way to do this.
`const { subtaskList = [] } = this.props;`

Reply:
Thanks for the advice! Your variant is definitely more readable and shorter =)